src/current/v23.1/crdb-internal.md

```diff
@@ -914,30 +914,30 @@ The `NumericStat` type tracks two running values: the running mean `mean` and th
 Field | Type | Description
 ------------|-----|------------
 `execution_statistics -> cnt` | `INT64` | The number of times execution statistics were recorded.
-<code>execution_statistics -> contentionTime -> [mean&#124;sqDiff]</code> | `NumericStat` | The time the statement spent contending for resources before being executed.
+<code>execution_statistics -> contentionTime -> [mean&#124;sqDiff]</code> | `NumericStat` | The time (in seconds) the statement spent contending for resources before being executed.
```

Comment:
I wonder if it is worth adding a Unit column rather than having it in the description, WDYT?

Reply:
Several rows do not have a unit, so another column would take up display space.